.github/workflows/deployment.yml

```diff
@@ -28,3 +29,14 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         BRANCH: gh-pages
         FOLDER: documentation
+    - name: Publish to npm
+      run: npm publish --access public
+      env:
+        NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+    - uses: actions/setup-node@v1
+      with:
+        registry-url: 'https://npm.pkg.github.com'
+    - name: Publish to GitHub Packages
```

Comment:
I would not publish it to GitHub packages too

Reply:
I'll just give it a try anyway. The neat thing is that this does not need an npm token, so things are more self-contained.